core/domain/learner_progress_services.py

```diff
@@ -913,6 +944,53 @@ def get_activity_progress(user_id):
         exploration_playlist_ids = []
         collection_playlist_ids = []
 
+    activity_ids = learner_progress_domain.ActivityIdsInLearnerDashboard(
+        completed_exploration_ids, completed_collection_ids,
+        incomplete_exploration_ids, incomplete_collection_ids,
+        exploration_playlist_ids, collection_playlist_ids)
+
+    return activity_ids
+
+
+def get_activity_progress(user_id):
+    """Returns the progress of the learners - the explorations and collections
+    completed by the user and those in progress.
+
+    Args:
+        user_id: str. The id of the learner.
+
+    Returns:
```

Comment:
I feel like I shouldn't have to comment on this in three separate passes, but there are still three return values here.

Reply:
Done!